webapp/app/css/treeherder.css

```diff
@@ -11,8 +11,8 @@ body {
     line-height: inherit; /* override bootstrap */
 }
 
-a:visited {
-    color: #a842ca;
+a:visited:not(.show-hide-more) {
```

Comment:
Squashed in a new commit that removes the show/hide selector. This PR now only changes visited link colors.